### PR TITLE
Added onnx test to RandomRGBShift

### PR DESCRIPTION
**Pull request by Oleksandra2020**
#### Changes

Fixes # (issue)

#### Type of change

- [ ] 📚  Documentation Update
- [ ] 🧪 Tests Cases
- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

**Review by edgarriba — APPROVED** [review]

**Comment by edgarriba** [comment]
@Oleksandra2020 you need to rebase properly with master. @shijianjian please, review

File: test/enhance/test_shift_rgb.py [hunk 1]
```diff
@@ -9,43 +9,43 @@
 class TestRGBShift:
 
     def test_rgb_shift_no_shift(self, device, dtype):
-        r_shift, g_shift, b_shift = 0, 0, 0
+        r_shift, g_shift, b_shift = torch.Tensor([0]), torch.Tensor([0]), torch.Tensor([0])
```
Comment (shijianjian) [inline test/enhance/test_shift_rgb.py:12]:
```suggestion
        r_shift, g_shift, b_shift = torch.tensor([0]), torch.tensor([0]), torch.tensor([0])
```

`torch.tensor` for creating torch instances, `torch.Tensor` is class types.

File: test/enhance/test_shift_rgb.py [hunk 2]
```diff
@@ -9,43 +9,43 @@
...
         image = torch.rand(2, 3, 5, 5, device=device, dtype=dtype)
         expected = image
         shifted = kornia.enhance.shift_rgb(image, r_shift, g_shift, b_shift)
 
         utils.assert_close(shifted, expected)
 
     def test_rgb_shift_all_zeros(self, device, dtype):
-        r_shift, g_shift, b_shift = -0.1, -0.1, -0.1
+        r_shift, g_shift, b_shift = torch.Tensor([-0.1]), torch.Tensor([-0.1]), torch.Tensor([-0.1])
```
Comment (shijianjian) [inline test/enhance/test_shift_rgb.py:20]:
```suggestion
        r_shift, g_shift, b_shift = torch.tensor([-0.1]), torch.tensor([-0.1]), torch.tensor([-0.1])
```

File: test/enhance/test_shift_rgb.py [hunk 3]
```diff
@@ -9,43 +9,43 @@
...
         image = torch.zeros(2, 3, 5, 5, device=device, dtype=dtype)
         expected = image
         shifted = kornia.enhance.shift_rgb(image, r_shift, g_shift, b_shift)
 
         utils.assert_close(shifted, expected)
 
     def test_rgb_shift_all_ones(self, device, dtype):
-        r_shift, g_shift, b_shift = 1, 1, 1
+        r_shift, g_shift, b_shift = torch.Tensor([1]), torch.Tensor([1]), torch.Tensor([1])
```
Comment (shijianjian) [inline test/enhance/test_shift_rgb.py:28]:
```suggestion
        r_shift, g_shift, b_shift = torch.tensor([1]), torch.tensor([1]), torch.tensor([1])
```

File: test/enhance/test_shift_rgb.py [hunk 4]
```diff
@@ -9,43 +9,43 @@
...
         image = torch.rand(2, 3, 5, 5, device=device, dtype=dtype)
         expected = torch.ones(2, 3, 5, 5, device=device, dtype=dtype)
         shifted = kornia.enhance.shift_rgb(image, r_shift, g_shift, b_shift)
 
         utils.assert_close(shifted, expected)
 
     def test_rgb_shift_invalid_parameter_shape(self, device, dtype):
-        r_shift, g_shift, b_shift = 0.5, 0.5, 0.5
+        r_shift, g_shift, b_shift = torch.Tensor([0.5]), torch.Tensor([0.5]), torch.Tensor([0.5])
```
Comment (shijianjian) [inline test/enhance/test_shift_rgb.py:36]:
```suggestion
        r_shift, g_shift, b_shift = torch.tensor([0.5]), torch.tensor([0.5]), torch.tensor([0.5])
```

File: test/enhance/test_shift_rgb.py [hunk 5]
```diff
@@ -9,43 +9,43 @@
...
         image = torch.randn(3, 3, device=device, dtype=dtype)
         with pytest.raises(TypeError):
             kornia.enhance.shift_rgb(image, r_shift, g_shift, b_shift)
 
     def test_rgb_shift_gradcheck(self, device, dtype):
-        r_shift, g_shift, b_shift = 0.4, 0.5, 0.2
+        r_shift, g_shift, b_shift = torch.Tensor([0.4]), torch.Tensor([0.5]), torch.Tensor([0.2])
```
Comment (shijianjian) [inline test/enhance/test_shift_rgb.py:42]:
```suggestion
        r_shift, g_shift, b_shift = torch.tensor([0.4]), torch.tensor([0.5]), torch.tensor([0.2])
```

File: test/enhance/test_shift_rgb.py [hunk 6]
```diff
@@ -9,43 +9,43 @@
...
         image = torch.randn(2, 3, 5, 5, device=device, dtype=dtype)
         image = utils.tensor_to_gradcheck_var(image)  # to var
         assert gradcheck(kornia.enhance.shift_rgb, (image, r_shift, g_shift, b_shift), raise_exception=True)
 
     def test_rgb_shift(self, device, dtype):
-        r_shift, g_shift, b_shift = 0.1, 0.2, -0.3
+        r_shift, g_shift, b_shift = torch.Tensor([0.1]), torch.Tensor([0.2]), torch.Tensor([-0.3])
```
Comment (shijianjian) [inline test/enhance/test_shift_rgb.py:48]:
```suggestion
        r_shift, g_shift, b_shift = torch.tensor([0.1]), torch.tensor([0.2]), torch.tensor([-0.3])
```

**Review by shijianjian — APPROVED** [review]
LGTM. Can you rebase and force push?

**Comment by edgarriba** [comment]
@Oleksandra2020 rebase went wrong -- i suggest to 
`git pull --rebase origin master`

**Comment by ducha-aiki** [comment]
@Oleksandra2020 I suggest to open new clean PR

**Pull request closed without merging**